File: Assets/ShootingRPGKit/Scripts/BulletAttackSkill/BulletChildGetDamage.js
#pragma strict
var master : GameObject;
var damagePercent : int = 100;

function Start () {
	if(!master){
		master = transform.root.gameObject;
	}
	var dmg : int = master.GetComponent(BulletStatus).totalDamage;
	
	if(damagePercent != 100){
		dmg = dmg * damagePercent / 100;
	}
	GetComponent(BulletStatus).totalDamage = dmg;
	GetComponent(BulletStatus).shooterTag = master.GetComponent(BulletStatus).shooterTag;
	GetComponent(BulletStatus).shooter = master.GetComponent(BulletStatus).shooter;
	
}

File: Assets/ShootingRPGKit/CsharpExample/ScriptCS/BulletAttackSkill/BulletChildGetDamage.cs
using UnityEngine;
using System.Collections;

public class BulletChildGetDamage : MonoBehaviour {
	public GameObject master;
	
	void  Start (){
		GetComponent<BulletStatus>().totalDamage = master.GetComponent<BulletStatus>().totalDamage;
		GetComponent<BulletStatus>().shooterTag = master.GetComponent<BulletStatus>().shooterTag;
		
	}
	
}